### ISSUE TITLE
Butterworth Kernel?

### ISSUE THREAD

**Issue by babaq** [JuliaImages/Images.jl#970]
Hi,

I can't find a butterworth kernel in the Images, and think would be a nice addition to the package.

The `DSP.jl` do have a Butterworth filter design, maybe could be used or looked up to generate the Factored Kernels. 

Alex

**Comment by johnnychen94** [comment]
Yes, it's a good addition, can you open a PR to ImageFiltering? That's where most the kernel lives.

**Comment by babaq** [comment]
I would like to contribute, but may not have the technical competence to implement the kernel, at least in a short period of time.

**Comment by johnnychen94** [comment]
You can take a look at https://github.com/JuliaImages/ImageFiltering.jl/pull/147 as an example. Some source codes, docstrings, and test cases should be sufficient.
